Bottom: b08c0727728eb3d5f8449ae6c754af58d7da154f
Top:    7f678b145131cca92f3d4ecf9503c34826087533
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-10 13:31:16 +0000

thermal.c: Add #warning pointing out where we might get a speedup






---

diff --git a/src/thermal.c b/src/thermal.c
index 445b97d..fa87f28 100644
--- a/src/thermal.c
+++ b/src/thermal.c
@@ -432,6 +432,7 @@ ThermPoly (PCBTypePtr p, PinTypePtr pin, Cardinal laynum)
         arc = CirclePoly (pin->X, pin->Y, pin->Thickness / 2);
         /* create a thin ring */
         poly_Boolean_free (pa, arc, &m, PBO_SUB);
+#warning THIS WOULD BE FASTER IF WE JUST SUTRACTED A CROSS SHAPE
         /* fix me needs error checking */
         if (style == 2)
           {
